docs/glossary.md

```diff
@@ -242,6 +254,12 @@ When service to the primary Cluster is compromised, a [failover](#failover) tran
 
 _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
+#### [Handler](/nexus#handler)
```

Comment:
would skip this one and keep the Nexus Operation Handler definition lower in the glossary.

```suggestion
```

Reply:
removing